addons/web/static/tests/_framework/touch_helpers.js

```diff
@@ -0,0 +1,64 @@
+import { queryFirst } from "@odoo/hoot-dom";
+import { contains } from "./dom_test_helpers";
+
+async function swipe(target, selector, direction, optionsDrag = {}, optionsMoveTo = {}) {
+    const touchTarget = queryFirst(selector, { root: target });
+    if (direction === "left") {
+        // The scrollable element is set at its right limit
+        touchTarget.scrollLeft = touchTarget.scrollWidth - touchTarget.offsetWidth;
+    } else {
+        // The scrollable element is set at its left limit
+        touchTarget.scrollLeft = 0;
+    }
+
+    const { moveTo, drop } = await contains(touchTarget).drag(
+        Object.assign(
+            {
+                position: {
+                    clientX: 0,
+                    clientY: 0,
+                },
+            },
+            optionsDrag
+        )
+    );
+    await moveTo(
+        touchTarget,
+        Object.assign(
+            {
+                position: {
+                    clientX: (direction === "left" ? -1 : 1) * touchTarget.clientWidth,
+                    clientY: 0,
+                },
+            },
+            optionsMoveTo
+        )
+    );
+    await drop();
+}
+
+/**
+ * Will simulate a swipe right on the target element with the given selector.
+ *
+ * @param {HTMLElement} target
+ * @param {DOMSelector} [selector]
+ * @param optionsDrag
+ * @param optionsMoveTo
+ * @returns {Promise}
+ */
+export async function swipeRight(target, selector, optionsDrag, optionsMoveTo) {
+    return swipe(target, selector, "right");
+}
+
```

Comment:
```suggestion
/**
 * Will simulate a swipe right on the target element with the given selector.
 *
 * @param {HTMLElement} target
 * @param {DOMSelector} [selector]
 * @returns {Promise}
 */
export async function swipeRight(target, selector) {
    return swipe(target, selector, "right");
}

```

Remove unused args.
Also why not use directly the `swipe` function instead of creating two new function for only setting the left and the right :thinking:

Reply:
Because it’s a conversion of a legacy test helper: https://github.com/odoo/odoo/commit/156d41dbda72ade4da13a6fcf384ba2164988bbc#diff-e2935dd99cea2688372cf9e72c432dc9567d2fce616985b849683ef9bcd5bc76R1